pandas/io/pytables.py

```diff
@@ -984,6 +984,7 @@ def put(
         data_columns: Optional[List[str]] = None,
         encoding=None,
         errors: str = "strict",
+        track_times: bool = True,
```

Comment:
can you update the doc-string, make sure to add a 1.1 versionadded tag

Reply:
doc string added